api-specificatie/BRP-Wijzigingen/openapi.yaml

```diff
@@ -0,0 +1,146 @@
+openapi: 3.0.0
+info:
+  title: "BRP Wijzigingen API"
+  version: "1.0.0"
+  contact:
+    name: test
+    email: me@home.nl
+    url: http://home.nl
+paths:
+  /wijzigingen:
+    get:
+      tags: 
+      - wijzigingen
+      operationId: getWijzigingen
+      parameters:
+        - name: afnemersindicatie
+          in: query
+          description: afnemersindicatie
+          required: true
+          schema:
+            type: string
+        - name: van
+          in: query
+          description: "van (yyyy-mm-ddThh:mm:ss)"
+          required: true
+          schema:
+            type: string
+            format: datetime
+        - name: tot
+          in: query
+          description: "tot (yyyy-mm-ddThh:mm:ss)"
+          required: false
+          schema:
+            type: string
+            format: datetime
+      responses:
+        200:
+          description: "response"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/WijzigingenHal'
+              examples:
+                gewijzigdIngeschrevenPersoon:
+                  $ref: '#/components/examples/GewijzigdIngeschrevenPersoon'
+                langerDanEenWeek:
+                  $ref: '#/components/examples/BevraagPeriodeLangerDanEenWeek'
+                alleenVan:
+                  $ref: '#/components/examples/AlleenVanDatumTijd'
+        400:
+          $ref: '#/components/responses/400'
+
+components:
+  responses:
+    '400':
+      description: Bad Request
+      content:
+        application/problem+json:
+          schema:
+            $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/BadRequestFoutbericht'
+          examples:
+            alleenTotDatum:
+              $ref: '#/components/examples/AlleenTotDatumTijd'
+
+  schemas:
+    WijzigingenHal:
+      type: object
+      properties:
+        _links:
+          $ref: '#/components/schemas/WijzigingenHal_links'
+        wijzigingen:
+          $ref: '#/components/schemas/Wijzigingen'
+    WijzigingenHal_links:
+      allOf:
+        - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalCollectionLinks'
+        - properties:
+            ingeschrevenpersoon:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink'
+            vervolgbevragingen:
+              type: array
+              items:
+                $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink'
+    Wijzigingen:
+      type: array
+      items:
+        $ref: '#/components/schemas/Wijziging'
+    Wijziging:
+      type: object
+      properties:
+        burgerservicenummer:
+          type: string
+        type:
```

Comment:
De type property is hier niet nodig. Er is maar één soort persoon binnen BRP in tegenstelling tot BRK waar je (ingeschreven) NP's & NNP's kan hebben

Reply:
We hebben voorlopig voorzien in deze property omdat het handig zou kunnen zijn als we dit mechanisme ook voor het uitvragen van rni willen gebruiken. Die vraag moet beantwoord worden.
In https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/623 heb ik deze vraag dan ook gesteld.

Reply:
ah ok. Die heb ik gemist. Maar hebben personen in rni een bsn? Of wordt daar een andere identificatie gebruikt? Want in dat geval zou ik burgerservicenummer hernoemen naar identificatie. Dan zou deze yaml ook voor rni kunnen worden hergebruikt.

Reply:
Helemaal gelijk maar wat ik uit [dit document](https://www.rvig.nl/binaries/rvig/documenten/brochures/2018/07/02/inschrijven-in-de-registratie-niet-ingezetenen/Brochure_inschrijven+in+de+RNI_NL.pdf) opmaak is dat personen die in de RNI worden opgenomen ook een bsn krijgen.
Ik wacht even totdat Cathy besloten heeft of ze met deze API ook de RNI wil uitvragen en pas daarna evt. de API aan.